modules/search/instant-search/components/search-app.jsx

```diff
@@ -116,27 +110,20 @@ class SearchApp extends Component {
 		} else {
 			this.hideResults();
 		}
-		this.getResults(
-			getSearchQuery(),
-			getFilterQuery(),
-			getSortQuery(),
-			getResultFormatQuery(),
-			null
-		);
+		this.getResults();
 	};
 
 	loadNextPage = () => {
-		this.hasNextPage() &&
-			this.getResults(
-				getSearchQuery(),
-				getFilterQuery(),
-				getSortQuery(),
-				getResultFormatQuery(),
-				this.state.response.page_handle
-			);
+		this.hasNextPage() && this.getResults( { pageHandle: this.state.response.page_handle } );
 	};
 
-	getResults = ( query, filter, sort, resultFormat, pageHandle ) => {
+	getResults = ( {
+		query = getSearchQuery(),
+		filter = getFilterQuery(),
+		sort = getSortQuery(),
+		resultFormat = getResultFormatQuery(),
+		pageHandle,
+	} ) => {
```

Comment:
```suggestion
	} = {} ) => {
```
Currently, `getResults` errors out when invoked without an input parameter (e.g. inside `onChangeQueryString`):

<img width="457" alt="Screen Shot 2019-11-08 at 2 28 01 PM" src="https://user-images.githubusercontent.com/4044428/68511818-fd087900-0233-11ea-83af-78d21a43bae7.png">